cfme/tests/intelligence/chargeback/test_chargeback_long_term_rates.py

```diff
@@ -290,10 +284,10 @@ def add_rate(tiered_rate):
                 cost = ((d['variable_rate'] * usage) + (d['fixed_rate'] * consumed_hours)) / 24
```

Comment:
**OPTIONAL** as far as i an tell the formula is always the same, can we look up the divisor in a mapping